Orange/evaluation/testing.py

```diff
@@ -8,7 +8,7 @@
 import numpy as np
 
 import joblib
-import sklearn.cross_validation as skl_cross_validation
+import sklearn.model_selection as skl_model_selection
```

Comment:
What about importing this as "skl"? As it is used a lot, a shorter name might be better. The name "skl.StratifiedKFold" signifies nicely that it is the scikit-learn class being used.